tests/docs/smoke-all/2024/07/18/10324.qmd

```diff
@@ -0,0 +1,56 @@
+---
+format: pdf
+keep-tex: true
+_quarto:
+  tests:
+    latex:
+      ensureFileRegexMatches:
+        - ['\\subcaption{\\label{tbl-example-2}Pressure}']
+        - []
+---
+
+::: {#tbl-example .cell layout-ncol="2" tbl-cap='Example' tbl-subcap='["Cars","Pressure"]'}
+
+```{.r .cell-code}
+library(tinytable)
+tt(head(cars)) |> theme_tt("tabular") |> 
+  save_tt("latex") |> knitr::asis_output()
```

Comment:
This line is superfluous. It was given in a reprex to match another example, but `save_tt()` and `knitr::asis_output()` should be called automatically by `tinytable`.

Reply:
Got it, and I'll make sure to use that in the future. But note that in this case this is not executable code anyway, it's the _output_ from knitr. We do this in our test suite so we don't have to worry about installing as many dependencies.